_includes/v21.2/misc/session-vars.html

```diff
@@ -839,5 +839,16 @@
    <td>Yes</td>
   </tr>
 
+  <tr>
+   <td>
+    <code>troubleshooting_mode_enabled</code>
+   </td>
+   <td>When enabled, avoid performing additional work on queries, particularly when the cluster is experiencing issues, unavailability, or failure. This session variable is used to avoid collecting and emitting telemetry data.</td>
```

Comment:
I think this is technically more correct:

```suggestion
   <td>When enabled, avoid performing additional work on queries, such as collecting and emitting telemetry data. This session variable is particularly useful when the cluster is experiencing issues, unavailability, or failure.</td>
```